### PR TITLE
Change travis-ci to github action

### PR DESCRIPTION
**Pull request by HJianBo**

### CODE REVIEW

File: .github/workflows/run_test_case.yaml [hunk 1]
```diff
@@ -18,6 +18,10 @@ jobs:
             make eunit
             make ct
             make cover
+        - name: Send coveralls
+          env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          run: rebar3 as test coveralls send
```
Comment (spring2maz) [inline .github/workflows/run_test_case.yaml]: remove the `coveralls` Makefile target ?

File: .github/workflows/run_test_case.yaml [hunk 2]
```diff
@@ -18,6 +18,11 @@ jobs:
             make eunit
             make ct
             make cover
+        - uses: coverallsapp/github-action@master
+        - name: Send coveralls
+          env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          run: rebar3 coveralls send
```
Comment (spring2maz) [inline .github/workflows/run_test_case.yaml]: or just `make coveralls`

File: .github/workflows/run_test_case.yaml [hunk 3]
```diff
@@ -18,6 +18,11 @@ jobs:
             make eunit
             make ct
             make cover
+        - uses: coverallsapp/github-action@master
```
Comment (spring2maz) [inline .github/workflows/run_test_case.yaml]: bad format. delete the leading dash `-`, and move it below `-name`
```
    - name: Coveralls
      uses: coverallsapp/github-action@master
      with:
        github-token: ${{ secrets.GITHUB_TOKEN }}
```

File: rebar.config [hunk 4]
```diff
@@ -31,13 +31,12 @@
...
 {cover_export_enabled, true}.
 
-{plugins, [coveralls]}.
-
 {erl_first_files, ["src/emqx_logger.erl"]}.
 
 {profiles,
  [{test,
-   [{deps,
+   [{plugins, [{coveralls, {git, "https://github.com/emqx/coveralls-erl", {branch, "github"}}}]},
```
Comment (spring2maz) [inline rebar.config]: Good job!
is this the fix ? https://github.com/emqx/coveralls-erl/commit/07c21d0ca08eacb0727da82c9a6c07437307fccc
maybe consider sending it as a PR to upstream.

File: rebar.config.script [hunk 5]
```diff
@@ -1,11 +1,21 @@
...
-                   {coveralls_service_name , "travis-ci"} | CONFIG];
-              _ ->
-                  CONFIG
-          end,
+%% -*-: erlang -*-
 
-CONFIG1.
+case {os:getenv("GITHUB_ACTIONS"), os:getenv("GITHUB_TOKEN")} of
+  {"true", Token} when is_list(Token) ->
+    Env = fun(K) -> os:getenv(K) end,
```
Comment (spring2maz) [inline rebar.config.script]: nit:  this only shortens the the call from `os:getenv` to `Env`, really necessary ?

**Comment by spring2maz** [comment]
Good job!
Please squash the commits.
Or if you plan to 'squash merge'? then consider it approved.

File: rebar.config (hunk 4 above)
Comment (HJianBo) [reply rebar.config]: Yes, Good idea :)

**Comment by HJianBo** [comment]
Thanks for your review here @spring2maz :]

I have made changes to follow your suggestion and squash all commits.

**Pull request merged**